src/bench/rpc_blockchain.cpp

```diff
@@ -53,7 +55,8 @@ BENCHMARK(BlockToJsonVerbose);
 static void BlockToJsonVerboseWrite(benchmark::Bench& bench)
 {
     TestBlockAndIndex data;
-    auto univalue = blockToJSON(data.block, &data.blockindex, &data.blockindex, *llmq::chainLocksHandler, *llmq::quorumInstantSendManager, /*verbose*/ true);
+    LLMQContext& llmq_ctx = *data.test_setup.m_node.llmq_ctx;
```

Comment:
missing const here
`const LLMQContext& llmq_ctx = *data.test_setup.m_node.llmq_ctx;`

Reply:
Resolved in latest push.